debezium-core/src/main/java/io/debezium/relational/history/DatabaseHistory.java

```diff
@@ -45,6 +45,16 @@
                     + "which it cannot parse. If skipping is enabled then Debezium can miss metadata changes.")
             .withDefault(false);
 
+    public static final Field STORE_ONLY_MONITORED_TABLES_DDL = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "store.only.monitored.tables.ddl")
+            .withDisplayName("Store only DDL that modifies whitelisted/not-blacklisted tables")
+            .withType(Type.BOOLEAN)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withDescription("Controls what DDL will Debezium store in history topic."
```

Comment:
... "in the DB history and schema change topics".

See my remark above, wondering whether you intentionally don't apply the option to the schema change topic. In that case some comment in the code would be nice (right now I think we should apply it to both topics).

Reply:
I dropped the word topic - as database history is generic and might not be implemented via Kafka